docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -255,12 +257,29 @@ the game ends in a draw:
       X | O | X
 
 
-XO Transaction Family Specification
-===================================
+Step 7. Delete the Game
+-----------------------
+
+Either user can use the ``xo delete`` command to remove the game from global
+state.
+
+.. code-block:: console
+
+   $ xo delete
```

Comment:
What's the difference between specifying the username and leaving it out?  Do these two commands do different things?

     xo delete my-game 

     xo delete my-game --username jack

Reply:
I meant this: `xo delete my-game --username jack --url http://rest-api:8008`

Reply:
Do you mean that --username is not optional?

Reply:
Just `xo delete` won't work

Reply:
I understand that the command must specify a game name.  My question is about the --username argument.  Is there a difference between these two commands? 

     xo delete my-game --url http://rest-api:8008

     xo delete my-game --username jack --url http://rest-api:8008

Reply:
if `--username` is not given,  the default `Linux user name` will be picked up.

Basically, same behavior as the `xo create game`

Reply:
Got it.  Thanks!!!

Reply:
delete only appears to care about the game name. I wouldn't mention `--username` as it's not going to do anything. Note I recently opened STL-1076 because delete is not specified.
`--url` will be relevant for all commands that are using the REST API under the hood.